glauth/pkg/command/server.go

```diff
@@ -62,11 +68,11 @@ func Server(cfg *config.Config) *cli.Command {
 
 				lcfg := glauthcfg.LDAP{
 					Enabled: cfg.Ldap.Enabled,
-					Listen:  cfg.Ldap.Address,
+					Listen:  cfg.Ldap.Addr,
```

Comment:
I am not getting the rational behind this change. It just makes stuff less readable (yeah, I know the golang mantra... ;-) and blows up this PR...

Reply:
The http package from the stdlib uses `.Addr`  and I got confused because in all other places we use `Addr` as well: https://github.com/owncloud/ocis/search?l=Go&q=Addr